src/frontends/tensorflow_lite/src/sparsity_info.cpp

```diff
@@ -14,22 +14,27 @@ bool ov::frontend::tensorflow_lite::SparsityInfo::is_copyable() const {
 template <typename T, typename U>
 static void read_sparse_data(uint8_t* dest,
                              uint8_t* dest_end,
+                             const uint8_t* values,
                              const size_t row_size,
+                             const size_t element_size,
                              const ::flatbuffers::Vector<T>* indices,
                              const ::flatbuffers::Vector<U>* segments) {
-    uint8_t* data = dest - row_size;  // row size will be increased at first step
-    T last_idx = ~static_cast<T>(0);
-    for (auto idx = indices->begin(); idx != indices->end(); ++idx) {
-        if (*idx <= last_idx) {
-            data += row_size;
+    U last_segment = *segments->begin();
```

Comment:
can you please elaborate a logic behind this? What is a segment, last segment, etc.?

Reply:
Maybe better to do it as a separate PR, because it will be just a comments. Let me check how many time I have.